### PR TITLE
JSDK-2087 Adding small group Room test.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@markandrus

### CODE REVIEW

File: test/integration/spec/rest.js [hunk 1]
```diff
@@ -20,26 +20,48 @@ const {
...
     request.end();
   });
+
+}
+
+function completeRoom(name) {
+  return post(`/v1/Rooms/${name}`, {
+    Status: 'completed'
+  });
+}
```
Comment (markandrus) [inline test/integration/spec/rest.js]: IMO, we should start using twilio-node. See how we can complete a Room [here](https://github.com/twilio/twilio-node/blob/master/lib/rest/video/v1/room.js#L690).

File: test/integration/spec/rest.js [hunk 2]
```diff
@@ -20,26 +20,48 @@ const {
...
+
+function createRoom(name, type) {
+  return post('/v1/Rooms', {
+    Type: type,
+    UniqueName: name
+  });
+}
```
Comment (markandrus) [inline test/integration/spec/rest.js]: IMO, we should start using twilio-node. See how we can create a Room [here](https://github.com/twilio/twilio-node/blob/master/lib/rest/video/v1/room.js#L86).

File: test/integration/spec/rest.js [hunk 3]
```diff
@@ -20,26 +20,48 @@ const {
...
+
+function subscribedTracks(publication, room, trackAction) {
+  const { localParticipant, sid } = room;
+  return post(`/v1/Rooms/${sid}/Participants/${localParticipant.sid}/SubscribedTracks`, {
+    Status: trackAction,
+    Track: publication.trackSid
+  });
```
Comment (markandrus) [inline test/integration/spec/rest.js]: IMO, we should start using twilio-node. See how we can subscribe to/unsubscribe from Tracks [here](https://github.com/twilio/twilio-node/blob/master/lib/rest/video/v1/room/roomParticipant/roomParticipantSubscribedTrack.js#L336).

File: test/integration/spec/rest.js (hunk 1 above)
Comment (manjeshbhargav) [reply test/integration/spec/rest.js]: @markandrus `twilio-node` does not seem to work in the browser due to node-specific dependencies.

File: test/integration/spec/rest.js (hunk 2 above)
Comment (manjeshbhargav) [reply test/integration/spec/rest.js]: @markandrus `twilio-node` does not seem to work in the browser due to node-specific dependencies.

File: test/integration/spec/rest.js (hunk 3 above)
Comment (manjeshbhargav) [reply test/integration/spec/rest.js]: @markandrus `twilio-node` does not seem to work in the browser due to node-specific dependencies.

**Comment by manjeshbhargav** [comment]
Re-opening against `master` here: https://github.com/twilio/twilio-video.js/pull/366

**Pull request closed without merging**